repos/base/lib/import/import-ada-lib.mk

```diff
@@ -0,0 +1,2 @@
+
+INC_DIR += $(REP_DIR)/src/test/ada-lib
```

Comment:
I'd prefer this called *test-ada* as it is used in the test case only.

Reply:
I would call this `test-ada-lib` then since `test-ada` is already the name of the test component.

Reply:
Sure.